ItemModCreationBoilerplate/Equipment/ExampleEquipment.cs

```diff
@@ -41,6 +31,11 @@ public override ItemDisplayRuleDict CreateItemDisplayRules()
             return new ItemDisplayRuleDict();
         }
 
+        protected override void Initialization()
```

Comment:
I assume the lack of an Init override in this example is to show that one can use the Initialization method without need to override Init?